graylog2-web-interface/src/stores/event-definitions/FilterPreviewStore.js

```diff
@@ -98,7 +106,7 @@ export const FilterPreviewStore = singletonStore(
           resolve(new SearchResult(job));
         } else {
           resolve(delay(250)
-            .then(() => this.jobStatus(job.id))
+            .then(() => this.jobStatus(job.id, job.executing_node))
```

Comment:
We can replace this with `runPollJob` from `views/stores/SearchJobs` to consolidate this.
Maybe we can even make use of `pollJob` from `executeJobResult` to make this consistent with our general search?

Reply:
I added runPollJob usage. We can refactor all of this and replace with a search implementation usage. But I would prefer to do it in separate PR.